### PR TITLE
Add CI workflows for main branch and tags

### PR DESCRIPTION
**Pull request by antonjahn**
Changes

* dropped elbe (layers + robot)
* fixed robot tests (docker -it does not work in CI, there is no interactive tty)
* renamed repo & image name, adjusted bash scripts in build/export/tests
* new workflow on main & prs > build, tests
* new workflow on tag > build, test, publish

Tested:
build and tests checked locally and in the ci job itself.
tag automation tested on fork, where I recreated the v1.3.9 branch

Question:
* Should we recreate the v1.3.9 tag to trigger the publish job or create a new v1.3.10 tag. The resulting image / features should be identical, as only config files, support scripts and tests changed.
* Recommendation: create v1.3.10 as we might need to troubleshoot the publish part in the Elektrobit org

### CODE REVIEW

File: builder/export_container
Comment (simone-weiss) [inline builder/export_container:17]: Should we use tooling like `yq` to read out the values from the YAML file?

Comment (antonjahn) [reply builder/export_container:17]: Hmm, your call.

It would be possible, see e.g. this commit & check https://github.com/antonjahn/ebcl_dev_container/commit/cd98ae548ece01420ba37c3cd5b295f32e336edc

| tool | Pro | Con |
|--------|--------|--------|
| awk | available on most linux boxes | syntax is not great |
| yq | can work with more complex yaml files, syntax is clean, is available on github runners | scripts are called locally and maintainers need to get yq as a dependency, okayish for modern ubuntu |

**Review by thir820 — APPROVED** [review]

**Review by simone-weiss — APPROVED** [review]

**Pull request merged**
